src/VisualStudio/Core/Def/Implementation/EditAndContinue/VsENCRebuildableProjectImpl.cs

```diff
@@ -30,6 +30,7 @@
 using ShellInterop = Microsoft.VisualStudio.Shell.Interop;
 using VsTextSpan = Microsoft.VisualStudio.TextManager.Interop.TextSpan;
 using VsThreading = Microsoft.VisualStudio.Threading;
+using Document = Microsoft.CodeAnalysis.Document;
```

Comment:
No, these are needed to avoid ambiguity. There is now Document type in System.Reflection.Metadata.